gitignore patch

Copyright (C) 2014 Curt Brune <curt@cumulusnetworks.com>

SPDX-License-Identifier:     GPL-2.0

Ignore files that are byproducts of the build process.

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..56dfaef
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,31 @@
+Makefile
+autom4te.cache/
+config.log
+config.status
+config/configure.in
+ct-ng
+docs/ct-ng.1
+docs/ct-ng.1.gz
+kconfig/conf
+kconfig/conf.dep
+kconfig/lex.backup
+kconfig/lex.zconf.c
+kconfig/lxdialog/checklist.dep
+kconfig/lxdialog/inputbox.dep
+kconfig/lxdialog/menubox.dep
+kconfig/lxdialog/textbox.dep
+kconfig/lxdialog/util.dep
+kconfig/lxdialog/yesno.dep
+kconfig/mconf
+kconfig/mconf.dep
+kconfig/nconf
+kconfig/nconf.dep
+kconfig/nconf.gui.dep
+kconfig/zconf.hash.c
+kconfig/zconf.tab.c
+kconfig/zconf.tab.dep
+paths.mk
+paths.sh
+scripts/crosstool-NG.sh
+scripts/saveSample.sh
+scripts/showTuple.sh
